orchagent/Makefile.am

```diff
@@ -25,6 +25,7 @@ endif
 
 orchagent_SOURCES = \
             main.cpp \
+            $(top_srcdir)/lib/gearboxutils.cpp \
```

Comment:
Does the `lib` directory need to be under `$(top_srcdir)`? Will this be shared by components other than gearsyncd?

Reply:
$ find . -print | xargs grep -I gearboxutils.h 2> /dev/null
./orchagent/main.cpp:#include "gearboxutils.h"
./orchagent/portsorch.h:#include "gearboxutils.h"
./orchagent/portsorch.cpp:#include "gearboxutils.h"
./orchagent/saihelper.h:#include "gearboxutils.h"
./gearsyncd/gearsyncd.cpp:#include "gearboxutils.h"

Reply:
I see. I was thinking it might be clearer to change those to `#include "lib/gearboxutils.h"`, but I'm OK with this for now.